mud-contracts/world/src/modules/smart-deployable/systems/SmartDeployable.sol

```diff
@@ -132,7 +137,11 @@ contract SmartDeployable is EveSystem, SmartDeployableErrors {
     _updateFuel(entityId);
     uint256 currentFuel = DeployableFuelBalance.getFuelAmount(_namespace().deployableFuelBalanceTableId(), entityId);
     if (currentFuel < 1) revert SmartDeployable_NoFuel(entityId);
-    DeployableFuelBalance.setFuelAmount(_namespace().deployableFuelBalanceTableId(), entityId, currentFuel - 1); //forces it to tick
+    DeployableFuelBalance.setFuelAmount(
+      _namespace().deployableFuelBalanceTableId(),
+      entityId,
+      currentFuel - (1 * (10 ** DECIMALS))
```

Comment:
We could store `10 ** DECIMALS` in a `ONE_FUEL_UNIT_IN_WEI` const to make the code more readable

Reply:
then for this one it'll be
```suggestion
      currentFuel - ONE_FUEL_UNIT_IN_WEI
```

(can also update all the others)

Reply:
fixed in [e72f25d](https://github.com/projectawakening/world-chain-contracts/pull/114/commits/e72f25dffac755c0c835ad7ac3f380d680994438)